src/java/org/apache/ivy/osgi/util/VersionRange.java

```diff
@@ -306,7 +306,7 @@ public boolean isClosedRange() {
         return startVersion.equals(endVersion);
     }
 
-    public boolean contains(String versionStr) throws ParseException {
+    public boolean contains(String versionStr) {
```

Comment:
Similar comment as here https://github.com/apache/ant-ivy/pull/52/files#r127395633 applies to this line